src/allotropy/allotrope/schemas/adm/electrophoresis/BENCHLING/2024/06/electrophoresis.schema.json

```diff
@@ -113,7 +113,7 @@
                                       "$asm.pattern": "quantity datum",
                                       "allOf": [
                                         {
-                                          "$ref": "http://purl.allotrope.org/json-schemas/adm/core/REC/2024/03/core.schema#/$defs/tQuantityValue"
+                                          "$ref": "#/$defs/tQuantityValue"
```

Comment:
Why this change?

Reply:
The `tQuantityValue` defined in the allotrope published `REC/2024/03/core.schema` does only support numeric values. Since we need to also support non numeric values ("NaN"), this change is necessary for the schema validation to pass.

Reply:
Ah gotcha 👍